cardano-cli/src/Cardano/CLI/EraBased/Run/Genesis/CreateTestnetData.hs

```diff
@@ -516,7 +516,9 @@ runGenesisCreateTestNetDataCmd
                     { L.drepExpiry = EpochNo 1_000
                     , L.drepAnchor = SNothing
                     , L.drepDeposit = max (L.Coin 1_000_000) minDeposit
-                    , L.drepDelegs = Set.empty
+                    , L.drepDelegs = Set.empty -- We don't need to populate this field (field "initialDReps"."keyHash-*"."delegators" in the JSON)
```

Comment:
Can you add the actual explanation from Alexy here and also confirm if the `"delegators" field will be removed?
Linking to the actual issue would be better: https://github.com/IntersectMBO/cardano-ledger/issues/4782#issuecomment-2523609735